### PR TITLE
Add dynamic description and keywords to Meta

### PR DESCRIPTION
**Pull request by lababidi**
This push allows authors to add a 'desc' and 'keywords' field to their post, which gets passed to the 'meta' tags on the html. The meta tags then get passed to search engines along with summarization engines such as Slack.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -3,8 +3,16 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    {% if page.keywords %}
```
Comment (geovations) [inline _layouts/default.html]: shouldn't this point at the page.keywords member, and not this list?

Comment (geovations) [inline _layouts/default.html]: Never mind. I see the {{ page.keywords }} at the end. Can you move it to the front to make it a bit more obvious?

**Comment by lababidi** [comment]
do not merge, still in draft.

File: _posts/2017-05-22-See-Beyond-Rainbow.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (geovations) [inline _posts/2017-05-22-See-Beyond-Rainbow.md]: Can you split this out?  This doesn't really go with the other stuff.

**Comment by lababidi** [comment]
@geovations should be back to normal

**Pull request merged**